src/containers/string.c

```diff
@@ -132,6 +132,27 @@ string_path_join(String* str1, String* str2, bool keep_strs) {
 }
 
 
+internal String*
+string_path_join_c_str(String* str1, const char* c_str, bool keep_strs) {
+    String* str2 = NULL;
+    String* str = NULL;
+    check(str1 != NULL, "str1 is NULL");
+    check(c_str != NULL, "c_str is NULL");
+    
+    // TODO: maybe add a string on stack??
```

Comment:
How can I do that?

Reply:
Its wierd with alloca and stuff, this shouldn't be critical so its fine for now at least